src/State/Inventory.cpp

```diff
@@ -308,6 +330,47 @@ void Inventory::onScrollDownButtonClick(Event::Mouse* event)
     if(inventory->canScrollDown())
     {
         inventory->scrollDown();
+        //enable/disable scroll buttons on downward scroll
+        if(!inventory->canScrollDown())
+        {
+            toggleScrollDownBtn(false);
+        }
+        if(inventory->canScrollUp())
+        {
+            toggleScrollUpBtn(true);
+        }
+    }
+}
+
+void Inventory::toggleScrollUpBtn(bool toggle)
```

Comment:
1) if that's "toggle", it should do exactly that, toggle, no parameters needed. Otherwise it should be named something like setEnabled or setState
2) please, don't abbreviate, Btn should be Button